### PR TITLE
web/admin: filter field for emulation options

### PR DESCRIPTION
**Pull request by mmadariaga**
Allow to filter brand / compny on emulation dialog

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: web/admin/application/languages/ca_ES/ca_ES.po [hunk 1]
```diff
@@ -3197,3 +3197,6 @@ msgstr "il·limitat"
 
 msgid "vPBX"
 msgstr "vPBX"
+
+#~ msgid "Select Company"
+#~ msgstr "Selecciona companyia"
```
Comment (Kaian) [inline web/admin/application/languages/ca_ES/ca_ES.po:3202]: Remove deprecated translation

File: web/admin/application/languages/en_US/en_US.po [hunk 2]
```diff
@@ -3166,3 +3166,6 @@ msgstr "unlimited"
 
 msgid "vPBX"
 msgstr "vPBX"
+
+#~ msgid "Select Company"
+#~ msgstr "Select Company"
```
Comment (Kaian) [inline web/admin/application/languages/en_US/en_US.po:3171]: Remove deprecated translation

File: web/admin/application/languages/es_ES/es_ES.po [hunk 3]
```diff
@@ -3209,3 +3209,6 @@ msgstr "ilimitado"
 
 msgid "vPBX"
 msgstr "vPBX"
+
+#~ msgid "Select Company"
+#~ msgstr "Seleccionar Compañía"
```
Comment (Kaian) [inline web/admin/application/languages/es_ES/es_ES.po:3214]: Remove deprecated translation

File: web/admin/application/languages/it_IT/it_IT.po [hunk 4]
```diff
@@ -3215,3 +3215,6 @@ msgstr "illimitato"
 
 msgid "vPBX"
 msgstr "vPBX"
+
+#~ msgid "Select Company"
+#~ msgstr "Seleziona Company"
```
Comment (Kaian) [inline web/admin/application/languages/it_IT/it_IT.po:3220]: Remove deprecated translation

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**